.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 python:
 - '2.7'
-env:
-- TOXENV=docs
-- TOXENV=py27
+- '3.5'
```

Comment:
I wanted to try to minimize the scope of the changes.  Once this is merged, adding 3.6 should be a breeze.